.github/workflows/beam_Publish_Beam_SDK_Snapshots.yml

```diff
@@ -45,6 +45,7 @@ env:
   GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
   GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
   GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+  docker_registry: us-central1-docker.pkg.dev
```

Comment:
Does this mean the docker image will publish to this registry? we currently publish to gcr.io/apache-beam-testing/ and internally rely on the snapshot for some testing purposes

Reply:
Thank you for noticing, I fixed it